### PR TITLE
PAINTROID-1_PAINTROID-14 Introduced Headless Retain Fragment instead …

### PR DESCRIPTION
**Pull request by DjordjeR**
…of Paintroid Application Singelton

### CODE REVIEW

File: Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java [hunk 1]
```diff
@@ -112,9 +112,9 @@
 	private boolean focusAfterRecreate = true;
 
 	public MainActivityPresenter(MainView view, Model model, Navigator navigator, Interactor interactor,
-			TopBarViewHolder topBarViewHolder, BottomBarViewHolder bottomBarViewHolder,
-			DrawerLayoutViewHolder drawerLayoutViewHolder,
-			NavigationDrawerViewHolder navigationDrawerViewHolder, CommandManager commandManager) {
+								TopBarViewHolder topBarViewHolder, BottomBarViewHolder bottomBarViewHolder,
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java]: Please revert unrelated style fixes to `MainActivityPresenter`

Comment (DjordjeR) [reply Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java]: I have reverted changes, for some reason Android Studio did this. Should I rename branch to only PAINTROID-14 and make a new request? I tought this solved the problem bacuse I tried going to the Import image from gallery ->  Hamburger menu -> Pocket paint and it no longer chrashes even though it did for play store version. Maybe this is another bug.

Comment (ThomasSchwengler) [reply Paintroid/src/main/java/org/catrobat/paintroid/presenter/MainActivityPresenter.java]: I will have another look at it, maybe this does fix our crashes already.

File: Paintroid/src/main/java/org/catrobat/paintroid/PaintroidApplicationFragment.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
+package org.catrobat.paintroid;
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/PaintroidApplicationFragment.java]: Please add our copyright header
